argoverse/data_loading/object_label_record.py

```diff
@@ -11,10 +11,22 @@
 import numpy as np
 
 from argoverse.utils.calibration import CameraConfig
-from argoverse.utils.cv2_plotting_utils import draw_clipped_line_segment
+from argoverse.utils.cv2_plotting_utils import draw_clipped_line_segment, proj_cam_to_uv
 from argoverse.utils.se3 import SE3
 from argoverse.utils.transform import quat2rotmat
 
+BKGRND_RECT_OFFS_UP = 30 # px
+BKGRND_RECT_OFFS_DOWN = 10 # px
+
+BKGRND_RECT_OFFS_LEFT = 70 # px
+BKGRND_RECT_OFFS_RIGHT = 70 # px
```

Comment:
Maybe a comment here explaining these? I'd probably have to do a little digging to know what they are.

Reply:
added a few comments, yeah that was not well-documented